scripts/zones/Uleguerand_Range/npcs/Rabbit_Footprint.lua

```diff
@@ -0,0 +1,109 @@
+-----------------------------------
+--  Area: Uleguerand_Range
+--   NPC: Rabbit Footprint (Spawns White/Black Coney)
+-----------------------------------
+local ID = require("scripts/zones/Uleguerand_Range/IDs")
+require("scripts/globals/status")
+require("scripts/globals/npc_util")
+-----------------------------------
+
+local points = {
+		[1]  = { 390.0830, -0.9039, -424.3834 },
+		[2]  = { 405.3988,  0.6603, -416.0932 },
+		[3]  = { 403.3210,  0.0126, -404.2754 },
+		[4]  = { 410.5687,  0.3688, -391.4490 },
+		[5]  = { 418.4933, -0.5737, -383.5276 },
+		[6]  = { 431.5144, -0.1639, -392.2122 },
+		[7]  = { 426.7340,  0.5218, -402.4513 },
+		[8]  = { 420.3062,  0.0522, -414.9328 },
+		[9]  = { 404.6560,  0.2487, -430.3242 },
+		[10] = { 411.2559,  0.3616, -441.8631 },
+		[11] = { 420.9601,  0.2481, -430.3242 },
+		[12] = { 427.6630,  0.4786, -434.4034 },
+		[13] = { 432.0447,  0.8000, -420.8208 },
+		[14] = { 442.7804,  0.3118, -410.2596 },
+		[15] = { 449.1314, -0.2384, -421.2825 },
+		[16] = { 443.1774,  0.1091, -433.7953 },
+		[17] = { 439.1841, -0.1640, -447.2572 }
+}
+
+function onTrade(player,npc,trade)
+    local coney = nil
+	local newMoon = IsMoonNew()
```

Comment:
only used once and doesn't change, does it need to be a variable for a reason I am not noticing?

Reply:
doesn't need to be a variable. bit of an artifact that i left that in as originally i was doing `local newMoon = VanadielMoonPhase() == 0 or false` and `local fullMoon = VanadielMoonPhase() == 4 or false` just as a nice readibility thing... before i realized there were actual functions i could use for new/full moon